README.md

```diff
@@ -306,6 +306,8 @@
 - [Sitesauce](https://sitesauce.app/) - A static version of your website in one click.
 - [Hello Sun](https://hellosun.brussels/) - Helping Brussels go solar.
 - [Nuxt](https://nuxtjs.org/) - Build your next Vue.js application with confidence with NuxtJS.
+- [OpenGraph](https://www.opengraph.xyz/) - Preview Open Graph Meta Tags for Your Website.
+- [ClinicNearMe.SG](https://www.clinicnearme.sg/) - Find 24-Hour Clinics, GPs, Hospitals & More in Singapore.
```

Comment:
These descriptions do not respect the conventions of the list. Do not use title case.